pymongo/server_description.py

```diff
@@ -243,8 +243,7 @@ def is_server_type_known(self) -> bool:
     def retryable_writes_supported(self) -> bool:
         """Checks if this server supports retryable writes."""
         return (
-            self._ls_timeout_minutes is not None
```

Comment:
Should we remove _ls_timeout_minutes and all code for it?

Reply:
Actually, according to the spec https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.rst, we still need to store the lowest `logicalSessionTimeoutMinutes` value in the `ToplogyDescription`.

Reply:
I see, so we still need to track it so we can use the lowest value for the server session cache expiration.

Reply:
Bingo.